src/ducks/account/helpers.js

```diff
@@ -106,8 +106,18 @@ export const getAccountType = account => {
   return type
 }
 
-export const getAccountInstitutionSlug = account =>
-  get(account, 'cozyMetadata.createdByApp')
+export const getAccountInstitutionSlug = account => {
+  const updatedByApps = get(account, 'cozyMetadata.updatedByApps')
+
+  if (Array.isArray(updatedByApps) && updatedByApps.length > 0) {
+    const sortedByDate = updatedByApps.sort(
+      (app1, app2) => new Date(app2.date) - new Date(app1.date)
+    )[0]
```

Comment:
I think we should use lodash's sortBy here.

1. Array::sort modifies the array in place, account is modified at the end of this method
2. lodash's sortBy has an easier to use API (not the principal reason)